Makefile

```diff
@@ -26,7 +41,7 @@ clean:
 # This Makefile requires CMake installed on the system
 .PHONY: $(OUTPUT)
 $(OUTPUT):
-	cmake -B build -S . && make -C build/ container -j6 && cp build/$(OUTPUT) $(OUTPUT)
+	cmake -B build -S . && cmake --build build --target $(NAME) --parallel 6 --config Release && cp $(OUTPUT_FILE) $(OUTPUT)
 
 .PHONY: test
 test: $(OUTPUT)
```

Comment:
For now `test` make target only works for linux since it expects to run go-worker tests too.